packages/paste-website/src/pages/form-elements/select/index.mdx

```diff
@@ -0,0 +1,944 @@
+
+import {Anchor} from '@twilio-paste/anchor';
+import {Box} from '@twilio-paste/box';
+import {Button} from '@twilio-paste/button';
+import {InformationIcon} from '@twilio-paste/icons/esm/InformationIcon';
+import {UnorderedList, ListItem} from '@twilio-paste/list';
+import {Callout, CalloutTitle, CalloutText} from '../../../components/callout';
+import {DoDont, Do, Dont} from '../../../components/DoDont';
+import {Codeblock} from '../../../components/codeblock';
+import {SidebarCategoryRoutes} from '../../../constants';
+import Changelog from '@twilio-paste/form/CHANGELOG.md';
+
+export const pageQuery = graphql`
+  {
+    allPasteComponent(filter: {name: {eq: "@twilio-paste/form"}}) {
+      edges {
+        node {
+          name
+          description
+          status
+          version
+        }
+      }
+    }
+    mdx(fields: {slug: {eq: "/form-elements/select/"}}) {
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<ComponentHeader
+  name="Select"
+  categoryRoute={SidebarCategoryRoutes.FORM}
+  githubUrl="https://github.com/twilio-labs/paste/tree/master/packages/paste-core/components/form"
+  storybookUrl="https://twilio-labs.github.io/paste/?path=/story/components-form--select-options"
+  abstractUrl="https://share.goabstract.com/6cc5fc40-3408-4cfd-b1d0-103aea01601d"
+  data={props.data.allPasteComponent.edges}
+/>
+
+---
+
+<contentwrapper>
+
+<TableOfContents headings={props.data.mdx.headings} />
+
+<content>
+
+## Guidelines
+
+A select allows users to select an option from a list in a form field.
+
+### About Select
+
+Select allows a user to make a selection from a list of options in a form field.
+
+By default, it will show the first option in the list. However, you can also set a different pre-selected option from your list to show.
+
+### What's the difference between select and a combobox?
+
+At its most basic, a select has an options list that’s styled according to the browser default. A combo box has a Twilio-styled options list and can allow for additional functionality like typeahead.
+
+Use a select when:
+
+<UnorderedList>
+  <ListItem>You need a native picker experience, especially on mobile devices.</ListItem>
+  <ListItem>
+    Users will be selecting from a list of 4-10 options, or a sorted list of highly familiar options (e.g., alphabetical
+    list of states or countries).
+  </ListItem>
+  <ListItem>You need the component to work out-of-the-box across all operating systems and browsers.</ListItem>
+</UnorderedList>
+
+Use a combobox when:
+
+<UnorderedList>
+  <ListItem>You need a Twilio-styled options list.</ListItem>
+  <ListItem>You need to show more than text in an option (e.g., text paired with an icon).</ListItem>
+  <ListItem>You need placeholder text in the field to show an example selection.</ListItem>
+  <ListItem>
+    Users would benefit from typeahead functionality (e.g., autocomplete, search). For example, typeahead may be useful
```

Comment:
Consider making this line "If you have a much longer list, consider lazy loading
    the list to improve page load performance." its own point "If you need a long list of options and thus would need to use lazy loading to improve page performance"